init.lua

```diff
@@ -91,7 +91,7 @@ vim.g.mapleader = ' '
 vim.g.maplocalleader = ' '
 
 -- Set to true if you have a Nerd Font installed and selected in the terminal
-vim.g.have_nerd_font = false
+vim.g.have_nerd_font = true
```

Comment:
I actually would prefer this being false. As i see it Kickstart project is about learning to configure stuff. If this is enabled by default it would:
1. Make people not go through the installation process of nerdfont
2. Make it harder to detect what is native part of neovim and what is not